lib/tasks/data_hygiene/publishing_delay_reporter.rb

```diff
@@ -0,0 +1,32 @@
+class Tasks::DataHygiene::PublishingDelayReporter
+  def report
+    now = Time.now
+
+    log_entries = ScheduledPublishingLogEntry.where(
+      :scheduled_publication_time.gt => now - 1.day,
+      :scheduled_publication_time.lte => now
```

Comment:
Mongoid will _probably_ allow us to write `.where(scheduled_publication_time: (now - 1.day)..now)`.  It _might_ be more efficient?  I am not a mongo expert by any means.

Reply:
Good shout. As far as I can tell it runs exactly the same query in Mongo, but it's sub-millisecond in either case.

Reply:
Oh, good to know it generates the same query.  Not that activerecord provides less-than or greater-than operators for where clauses, but I'm p. sure it wouldn't optimize two conditionals like this into a between query for you if it did.